src/node_interface.rs

```diff
@@ -63,28 +75,34 @@ pub trait Interface : Sync + Send {
                    name : NameType,
                    data: Vec<u8>) -> Result<MessageAction, InterfaceError>;
 
+    /// consumes data in response or handles the error
     fn handle_get_response(&mut self,
                            from_address: NameType,
                            response: Result<Vec<u8>, ResponseError>) -> MethodCall;
 
+    /// handles the result of a put request
     fn handle_put_response(&mut self,
                            from_authority: Authority,
                            from_address: NameType,
                            response: Result<Vec<u8>, ResponseError>) -> MethodCall;
 
+    /// handles the result of a post request
```

Comment:
Please do not use comments to repeat the function name. Comments should describe more than spelling out the name. Please see http://aturon.github.io/style/comments.html for rust and here and note the comments add more info as opposed to repeating the name of a function in English.